soc/arm/st_stm32/stm32l1/Kconfig.defconfig.stm32l151xb

```diff
@@ -0,0 +1,28 @@
+# Kconfig - ST Microelectronics STM32L151XB MCU
+#
+# Copyright (c) 2019 Linaro Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32L151XB
+
+config SOC
+	string
+	default "stm32l151xb"
+
+config NUM_IRQS
+	int
+	default 32
```

Comment:
Can you double check this value?
You're using IRQ num > 32 in dts, this should be causing the issue detected by shippable

Reply:
Ah. It is 45. Updated in next revision!